### PR TITLE
Bump: Kafkas brokers 1.2.3

### PR DESCRIPTION
**Pull request by matzew**
/hold

drafting the bump for updating the "brokers" bits

### CODE REVIEW

**Review by aliok — APPROVED** [review]
/lgtm
/approve

File: knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml [hunk 1]
```diff
@@ -1103,55 +1297,10 @@ spec:
...
-    # buckets is the number of buckets used to partition key space of each
-    # Reconciler. If this number is M and the replica number of the controller
-    # is N, the N replicas will compete for the M buckets. The owner of a
-    # bucket will take care of the reconciling for the keys partitioned into
-    # that bucket.
-    buckets: "1"
-  leaseDuration: "15s"
-  renewDeadline: "10s"
-  retryPeriod: "2s"
-  map-lease-prefix.kafka-broker-controller.knative.dev.eventing-kafka-broker.control-plane.pkg.reconciler.source.Reconciler: kafka-controller.knative.dev.eventing-kafka.pkg.source.reconciler.source.reconciler
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml:1157]: Why is this deleted?

/hold

File: knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml [hunk 2]
```diff
@@ -311,6 +436,55 @@ rules:
...
+
+    # buckets is the number of buckets used to partition key space of each
+    # Reconciler. If this number is M and the replica number of the controller
+    # is N, the N replicas will compete for the M buckets. The owner of a
+    # bucket will take care of the reconciling for the keys partitioned into
+    # that bucket.
+    buckets: "1"
+  leaseDuration: "15s"
+  renewDeadline: "10s"
+  retryPeriod: "2s"
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml:486]: We should include this line here [1] in this CM

[1] https://github.com/openshift-knative/serverless-operator/pull/1486/files#diff-d437e429fa5972ccdd7a526b258b9a4671f31e50a81d2a666d109c46252a631eL1157

File: knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml (hunk 1 above)
Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/controller/1-eventing-kafka-controller.yaml:1157]: @pierDipi why is there none for this, in upstream ?

**Comment by matzew** [comment]
/close

worked on via https://github.com/openshift-knative/serverless-operator/pull/1489
